Dockerfile

```diff
@@ -44,6 +44,8 @@ RUN pip install --no-cache-dir /src/scraper \
 # Copy zimui build output
 COPY --from=zimui /src/dist /src/zimui
 
-ENV LIBRETEXTS_ZIMUI_DIST=/src/zimui LIBRETEXTS_OUTPUT=/output LIBRETEXTS_TMP=/tmp
+ENV LIBRETEXTS_ZIMUI_DIST=/src/zimui \
```

Comment:
Not what I expected but it is better this way: visible and on a single stmt (although ENV stmt dont generate a layer each I believe)

Reply:
I'm never sure about layers, and tbh I first created multiple `ENV` stmt and then thought it was bad because it is anyway more readable this way.